library/koji_tag_inheritance.py

```diff
@@ -45,6 +45,12 @@
        - The priority of this parent for this child. Parents with smaller
          numbers will override parents with bigger numbers.
      required: true
+   maxdepth:
+     description:
+       - Maximum depth of the inheritance. For example "0" means that only
+         the parent tag itself will be available in the inheritance - parent
+         tags of the parent tag won't be available.
+     required: false
```

Comment:
Since this is `required: false`, would you please add a line that describes the default behavior if users do not set this parameter?

Is there any way for users to *unset* a maxdepth value on a tag that has maxdepth already configured? I'm wondering what the syntax would be. `maxdepth: ` (blank)?

Reply:
I don't want to make the option "required" as it would break existing playbooks and roles.

I checked and unset currently will happen when:
1) the option is not defined - I think this is not a behavior that one would expect from Ansible playbook (to change a state of maxdepth if the option is not defined - on the other hand, this is the current behavior of this koji-ansible library anyway).
2) The option is blank ``maxdepth: `` this has the same effect as not defining the option.

What do you think Ken?

Reply:
Thanks for clarifying this. I tested this out on my koji instance today and I see what you mean.

Let's merge this now so it's usable, and we can add more documentation in a separate PR.